README.rst

```diff
@@ -261,58 +259,10 @@ Run tests (including coverage) with:
 Conda package
 =============
 
-To build anaconda package locally, do:
-
-.. code-block:: console
-
-  conda deactivate
-  conda env create --file conda/environment-build.yml
-  conda activate matchms-build
-  BUILD_FOLDER=/tmp/matchms/_build
-  rm -rfv $BUILD_FOLDER;mkdir -p $BUILD_FOLDER
-  conda build --no-include-recipe -c bioconda -c conda-forge \
-  --croot $BUILD_FOLDER ./conda
-
-If successful, this will yield the built ``matchms`` conda package as
-``matchms-<version>*.tar.bz2`` in ``$BUILD_FOLDER/noarch/``. You can test if
-installation of this conda package works with:
-
-.. code-block:: console
-
-  # make a clean environment
-  conda deactivate
-  cd $(mktemp -d)
-  conda env create --name test python=3.7
-  conda activate test
-
-  conda install \
-    --channel bioconda \
-    --channel conda-forge \
-    --channel file://${CONDA_PREFIX}/noarch/ \
-    matchms
-
-To publish the package on anaconda cloud, do:
-
-.. code-block:: console
-
-  anaconda --token ${{ secrets.ANACONDA_TOKEN }} upload --user nlesc --force $BUILD_FOLDER/noarch/*.tar.bz2
-
-where ``secrets.ANACONDA_TOKEN`` is a token to be generated on the Anaconda Cloud website. This secret should be added to GitHub repository.
-
-
-To remove matchms package from the active environment:
-
-.. code-block:: console
-
-  conda remove matchms
-
-
-To remove matchms-build environment:
-
-.. code-block:: console
-
-  conda env remove --name matchms-build
+The conda packaging is handled by a `recipe at Bioconda <https://github.com/bioconda/bioconda-recipes/blob/master/recipes/matchms/meta.yaml>`_.
 
+Publishing to PyPI will trigger the creation of a `pull request on the bioconda recipes repository <https://github.com/bioconda/bioconda-recipes/pulls?q=is%3Apr+is%3Aopen+matchms>`_
+Once the PR is merged the new version of matchms will appear on `https://anaconda.org/bioconda/matchms <https://anaconda.org/bioconda/matchms>`_
```

Comment:
Didn't think of the readme part at all. Nice that it's now clear that the conda part has moved to Bioconda!